src/site/components/fullwidth_banner_alerts.drupal.liquid

```diff
@@ -0,0 +1,76 @@
+{% for entity in bannerAlert.entities %}
+  {% assign dismissStatus = entity.fieldAlertInheritanceSubpages %}
+  {% assign alertType = entity.fieldAlertType %}
+
+  {% if alertType = "information" %}
+    {% assign alertType = "info" %}
+  {% endif %}
+
+  {% assign region = path | split: "/" | first | prepend: "/" %}
+  {% assign lastArg = path | split: "/" | last | prepend: "/"%}
+
+  {% assign outputStatus = false %}
+  {% assign emailUpdates = false%}
+  {% assign statusUrl = false %}
+
+  {% for vamc in entity.fieldBannerAlertVamcs %}
+
+    {% if region == vamc.entity.fieldOffice.entity.entityUrl.path %}
+
+      {% assign outputStatus = true %}
+
+      {% if dismissStatus == true and lastArg != region and lastArg != "/operating-status" %}
+        {% assign outputStatus = false %}
+      {% endif %}
+      {% assign emailUpdates = vamc.entity.fieldOffice.entity.fieldLinkFacilityEmergList.uri %}
+      {% assign statusUrl = vamc.entity.entityUrl.path %}
+    {% endif %}
+  {% endfor %}
+
+  {% if outputStatus == true %}
+    <div aria-labelledby="usa-alert-heading-{{ entity.entityId }}"
+      class="usa-alert-full-width dismissable-option-header usa-alert-full-width-{{ alertType }}"
+      id="usa-alert-full-width-{{ entity.entityId }}" role="region">
+      <div class="usa-alert usa-alert-{{ alertType }}" role="alert">
+        {% if entity.fieldAlertDismissable == true %}
+          <button id="usa-alert-dismiss-{{ entity.entityId }}"
+            class="va-alert-close usa-alert-dismiss"
+            data-parentwrap="usa-alert-full-width-{{ entity.entityId }}"
+            data-frequency="once" aria-label="Close notification">
+            <i aria-label="Close icon" class="fas fa-times-circle"></i>
+          </button>
+        {% endif %}
+        <div class="usa-alert-body">
+          <h3 class="usa-alert-heading">
+            {{ entity.title }}
+          </h3>
+          <div class="additional-info-content usa-alert-text">
+            {{ entity.fieldBody.processed }}
+
+            {% if entity.fieldAlertOperatingStatusCta == true and statusUrl != false %}
```

Comment:
Sorry I am not well versed on strictness of mustache comparisons.   My thinking (which may be flawed) is that this is too strict a comparison looking for false.  It seems like it should need to compare to being 'not empty' as it is possible up on line 26 that statusUrl could end up empty from the node data, but that would replace the default of false.   

If you are sure that empty string and false both equate to false, then nevermind.

Reply:
Yeah, I didn't think about that array possibly being empty. I'll set the variables to an empty string, and do a length check (just in case setting to "false" somehow gets interpreted as truthy for length), @swirtSJW